keyboards/skyloong/gk980/bl/matrix.c

```diff
@@ -0,0 +1,166 @@
+// Copyright 2023 NaturaZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+//Matrix read rows set col:DIODE_DIRECTION == ROW2COL
+
+#include "quantum.h"
+#include "debounce.h"
+
+#define ReadDelayTime 120
+#define ClockTime 15
+#define MATRIX_INPUT_PRESSED_STATE 0
+
+#define HC595_ST_PIN A6
+#define HC595_SH_PIN A5
+#define HC595_DS_PIN A7
+
+/* matrix state(1:on, 0:off) */
+extern matrix_row_t raw_matrix[MATRIX_ROWS]; // raw values
+extern matrix_row_t matrix[MATRIX_ROWS];     // debounced values
+
```

Comment:
```suggestion
```

Reply:
I wish I could debounce the keys.

Reply:
The debouncing is handled for you when using custom matrix lite. PLEASE READ THE DOCS.